library/xwiki

```diff
@@ -2,36 +2,36 @@ Maintainers: XWiki Core Dev Team <committers@xwiki.org> (@xwiki)
 GitRepo: https://github.com/xwiki-contrib/docker-xwiki.git
 
 # Stable
-Tags: 16, 16.10, 16.10.2, 16-mysql-tomcat, 16.10-mysql-tomcat, 16.10.2-mysql-tomcat, mysql-tomcat, lts-mysql-tomcat, lts-mysql, lts, stable-mysql-tomcat, stable-mysql, stable, latest
+Tags: 17, 17.0, 17.0.0, 17-mysql-tomcat, 17.0-mysql-tomcat, 17.0.0-mysql-tomcat, mysql-tomcat, stable-mysql-tomcat, stable-mysql, stable, latest
 Architectures: amd64, arm64v8
-Directory: 16/mysql-tomcat
-GitCommit: beed20b75ae4c952be757cec1502151d2d2a7939
+Directory: 17/mysql-tomcat
+GitCommit: 381926879a24fb09d3b7611b883c78eb4576d3a2
 
-Tags: 16-postgres-tomcat, 16.10-postgres-tomcat, 16.10.2-postgres-tomcat, postgres-tomcat, lts-postgres-tomcat, lts-postgres, stable-postgres-tomcat, stable-postgres
+Tags: 17-postgres-tomcat, 17.0-postgres-tomcat, 17.0.0-postgres-tomcat, postgres-tomcat, stable-postgres-tomcat, stable-postgres
 Architectures: amd64, arm64v8
-Directory: 16/postgres-tomcat
-GitCommit: beed20b75ae4c952be757cec1502151d2d2a7939
+Directory: 17/postgres-tomcat
+GitCommit: 381926879a24fb09d3b7611b883c78eb4576d3a2
 
-Tags: 16-mariadb-tomcat, 16.10-mariadb-tomcat, 16.10.2-mariadb-tomcat, mariadb-tomcat, lts-mariadb-tomcat, lts-mariadb, stable-mariadb-tomcat, stable-mariadb
+Tags: 17-mariadb-tomcat, 17.0-mariadb-tomcat, 17.0.0-mariadb-tomcat, mariadb-tomcat, stable-mariadb-tomcat, stable-mariadb
 Architectures: amd64, arm64v8
-Directory: 16/mariadb-tomcat
-GitCommit: beed20b75ae4c952be757cec1502151d2d2a7939
+Directory: 17/mariadb-tomcat
+GitCommit: 381926879a24fb09d3b7611b883c78eb4576d3a2
 
-# old LTS
-Tags: 15, 15.10, 15.10.16, 15-mysql-tomcat, 15.10-mysql-tomcat, 15.10.16-mysql-tomcat
+# LTS
+Tags: 16, 16.10, 16.10.2, 16-mysql-tomcat, 16.10-mysql-tomcat, 16.10.2-mysql-tomcat, mysql-tomcat, lts-mysql-tomcat, lts-mysql, lts
```

Comment:
Duplicate tag here with `mysql-tomcat` that is also up in `17*` (line 5), so probably can be dropped from here.  This'll also need a rebase since I just merged https://github.com/docker-library/official-images/pull/18325.

Reply:
Addressed in e07eb642015f70bc8d7160cb117b098ec0bb1870